### ISSUE TITLE
Downloading in ORIGINAL size doesnot working.

### ISSUE THREAD

**Issue by KarelHaruda** [mistic100/Piwigo-BatchDownloader#10]
I have piwigo 2.8 and  BatchDownloader in version 2.2.2.

when I downloading album in XXL size it write information and start comprimation and show progress bar
Album: foto / 103 10

```
47 fotografií
Velikosti fotografií: Větší XXL (1656 x 1242)
Odhadnutá velikost: 26 MB
Odhadnutý počet archvů: 1 (opravdový počet archivů se může lišit)
Vytvořeno: Čtvrtek 20 Říjen 2016
```

BUT when I try it with ORIGINAL size i get information and no prograssbar and comprimation doesnot start and it is all .. (no error in log)
Album: foto / 103 10

```
47 fotografií, Upravit sadu
Velikosti fotografií: Originál
Odhadnutá velikost: 0 MB
Odhadnutý počet archvů: 0 (opravdový počet archivů se může lišit)
Vytvořeno: Čtvrtek 20 Říjen 2016
```

**Comment by KarelHaruda** [comment]
Original size of stored files is 3240x4320

**Comment by tjw1184** [comment]
I'm seeing this same behavior in piwigo 2.9.1 with BatchDownloader version 2.2.3.  Were you able to find a way to resolve it?

**Comment by KarelHaruda** [comment]
No, we have not resolved yes. We use  XXL  size to download, but we are not able to download in original size.

**Comment by rqchew** [comment]
Same issue with this. An album with 3 pictures downloaded fine at Original size though.

**Comment by KarelHaruda** [comment]
I have tested it on version 2.2.3 and I can confirm that now it works fine :) .. thank you ..

**Comment by rqchew** [comment]
Still seeing this behaviour when more than 1 .ZIP needs to be generated. The last .ZIP would n ot be generated in original size. I've gone around it by setting the maximum size to 500MB though.

**Comment by mistic100** [comment]
Not related with 2.2.3 which was only a translation update.

I already encountered this behavior and I am convinced this is related to some obscure server configuration. I don't know which.

**Comment by wurststulle** [comment]
someone managed to solve this issue?

**Comment by patrickg75** [comment]
Also having the same behavior with version 12a.
Originals are not downloadable. 🙁
